src/SFA.DAS.EmployerAccounts.Web/Controllers/EmployerAgreementController.cs

```diff
@@ -222,6 +222,11 @@ public async Task<ActionResult> ConfirmRemoveOrganisation(string accountLegalEnt
         {
             var model = await _orchestrator.GetConfirmRemoveOrganisationViewModel(accountLegalEntityHashedId, hashedAccountId, OwinWrapper.GetClaimValue(ControllerConstants.UserRefClaimKeyName));
 
+            if (model.Status == HttpStatusCode.Unauthorized)
```

Comment:
I think this should be handled by the base controller. I think the only change needed to be made is to pass `model` rather than `model.data` to the `View()`